polybar-scripts/info-todotxt/info-todotxt.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+duetoday=$(grep -c "$(date -I)" ~/todo.txt)
+dueweek=0
+
+for i in $(echo {0..7}day); do
```

Comment:
`shellcheck` fails here. for-loops are a bit hard in POSIX. As workaround we could use `while`.

Could you test something like this:
```
weekday=0
while [ "$weekday" -le 7 ]; do
    dueweek=$((dueweek + $(grep -c "$(date -I --date="$weekday")" ~/todo.txt)))
    weekday=$(( weekday + 1 ))
done
```